.neuro/live.yaml

```diff
@@ -0,0 +1,51 @@
+kind: live
+title: Horovod example
+id: horovod_example
+
+volumes:
+  src:
+    remote: storage:${{ flow.flow_id }}/src
+    mount: /root/app/src
+    local: src
+  results:
+    remote: storage:${{ flow.flow_id }}/results
+    mount: /root/app/results
+
+images:
+  horovod:
+    ref: horovod/horovod:sha-52d0b27
+
+jobs:
+  main:
+    image: ${{ images.horovod.ref }}
+    pass_config: true
+    name: ${{ lower(replace(flow.title, " ", "-")) }}-train
+    volumes:
+      - secret:horovod-id-rsa:/root/.ssh/id_rsa
```

Comment:
just curious: does it have to be a separate key? that is — can we use same key name in all examples just for simplicity?